robot/rover/PDS/PDS.ino

```diff
@@ -179,108 +197,150 @@ void loop() {
       tempReadings[i] = (1.0 / (A + B * ln + C * pow(ln, 3))) - 273.15;
     }
   }
+
   /***************************************************************************************************************/
   if (errorFlagGeneral.UV) {
     errorFlagGeneral.criticalError = 1;
     PORTB &= 0B11111110; //motor one
     PORTD &= 0B00111111; //motor two and three
     PORTC &= 0B11000111; //motor four, five and six
   }
-  /***************************************************************************************************************/
-  if (!errorFlagGeneral.UV) {
-    // convert speed percentage to PWM based on battery voltage
-    MAX_FAN_SPEED = (int)(12 * 255 / batteryVoltage); // calculate the max allowable fan speed
-    fanASpeed = map(fanASpeed, 0, 255, 0, MAX_FAN_SPEED); // attempt to keep the pwm voltage at 12v
-    fanBSpeed = map(fanBSpeed, 0, 255, 0, MAX_FAN_SPEED); // attempt to keep the pwm voltage at 12v
-    analogWrite(Fan_A_Speed_Pin, fanASpeed);
-    analogWrite(Fan_B_Speed_Pin, fanBSpeed);
-  } else {
-    analogWrite(Fan_A_Speed_Pin, 0);
-    analogWrite(Fan_B_Speed_Pin, 0);
-  }
+
+    if (!errorFlagGeneral.UV) {
+      // ATTENTION: Needs fixing!!!!!!!!!!!!!!!!!!!!!
```

Comment:
You may want to open an issue or fix it so we don't forget.

Reply:
There's an issue for all the pds changes

Reply:
@vashmata @MartensCedric just a reminder for us that we need to touch base with ELEC and create an issue/issues for the corresponding [epic](https://github.com/space-concordia-robotics/robotics-prototype/issues/126)